apps/avifdec.c

```diff
@@ -79,7 +79,11 @@ int main(int argc, char * argv[])
     while (argIndex < argc) {
         const char * arg = argv[argIndex];
 
-        if (!strcmp(arg, "-h") || !strcmp(arg, "--help")) {
+        if (!strcmp(arg, "--")) {
```

Comment:
The `syntax()` function should document `--`.

Also in avifenc.c.

Note that we also have man pages for avifdec and avifenc. doc/avifdec.1.md and doc/ avifenc.1.md should document `--`.

Reply:
Done but note that this is a standard convention and not always documented. E.g. if you look at the man page of "cp" or "mv" they don't talk about -- but do support it.

Reply:
When I wrote this suggestion, I didn't know `--` is a well-known convention. If you think it is not necessary to document this well-known convention, please remove it from the `syntax()` functions and the man pages.

Reply:
I think it's fine either way. cwebp does document it. It doesn't hurt to have it in the doc.